languages/java11/template/src/main/java/com/algorithmia/algorithm/Algorithm.java

```diff
@@ -0,0 +1,35 @@
+package com.algorithmia.algorithm;
+
+import com.algorithmia.development.*;
+import com.algorithmia.*;
+
+import java.util.HashMap;
+
+
+// This class defines your algorithm.
+class Algorithm extends AbstractAlgorithm<String, String>{
+
+    // This class member can be defined as anything you wish, and stores loaded state and mutable state between requests.
+     HashMap<String, String> context = new HashMap<>();
+
+    // This apply function defines the primary motive driver of your algorithm. Please ensure that the types defined in
+    // your algorithm are the same as those defined in your class implementation above ^
+
+    public String apply(String input){
+        // ... = context.get("loaded_model_file")
```

Comment:
i think it would be better practice to just have a member variable `String loadedModelFileName` than to encourage people to store stuff in a hashmap

Reply:
Yeah that's simpler, I can do that